src/Component/Cart.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+function Cart ({cart, subtotal}) {
+    return (
+        <div className="">
+        <h2>Enrolled</h2>
+        <ul>
+            {cart.map((item)=>{
+                return <li>{item.name} ${item.price}</li>
+            })}
+        </ul>
+        <h3>Subtotal: 
+            {subtotal}
+            {/* {subtotal.reduce((acc,curr)=> acc + curr)} */}
+            </h3>
+        <h3>Tax: ${subtotal * .05}</h3>
+        <h3>Total:${(subtotal * .05) + subtotal}</h3>
```

Comment:
The tests expect each of these prices to be formatted with a dollar sign and two places after the decimal. You can use the helper function in `src/helpers/formatPrice` to assist with this.